src/applications/personalization/profile/tests/components/ProfileWrapper.unit.spec.js

```diff
@@ -9,9 +9,8 @@ import { PROFILE_PATHS } from '../../constants';
 import { renderWithProfileReducers as render } from '../unit-test-helpers';
 
 describe('ProfileWrapper', () => {
-  let ui;
   const config = {};
-  ui = (
```

Comment:
ohhhh I see now. this could have been put in a `beforeEach()` instead and the problem would be fixed.